### ISSUE TITLE
Add setting to always edit commit messages in centre pane

### ISSUE THREAD

**Issue by bennypowers** [atom/github#1093]
### Prerequisites

* Have you tried to reproduce the problem in [Safe Mode][safe-mode]? N/A
* Have you followed all applicable steps in the [debugging guide][debugging-guide]? N/A
* Have you checked the FAQs on the [message board][message-board] for common solutions? Yes
* Have you checked that your issue [isn't already filed][already-filed]? Searched through old issues, didn't find dupe.

[safe-mode]: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
[debugging-guide]: http://flight-manual.atom.io/hacking-atom/sections/debugging/
[message-board]: https://discuss.atom.io/c/faq
[already-filed]: https://github.com/atom/github/issues

### Description

1.20 provides the option to edit commit messages in the centre pane. I'd like a toggle switch in `github` settings which will configure the plugin to always edit commit messages in centre pane.

A command to make a commit could then be bound to a hotkey, and that would open the pane.

### Versions

^1.2.0

### Additional Information

The motivation for this issue is to be able to use Vim Mode Plus to edit commit messages.

**Comment by bronson** [comment]
Another motivation is to be able to use a [git commit message grammar](https://github.com/atom/language-git/blob/master/grammars/git%20commit%20message.cson).

This is the most effective way that I've seen to cut down on crappy commit messages.  Instant feedback is powerful.

**Comment by BinaryMuse** [comment]
@bennypowers There is a command called `github:toggle-expanded-commit-message-editor` that you can use to open the commit message editor. Does this do what you're looking to do or is there something more?

**Comment by bennypowers** [comment]
@BinaryMuse yes that's exactly what I'd like to do. What I'd like is to fire that command every time the non-expanded editor is focused.
